src/private/interfaces.ts

```diff
@@ -43,6 +43,23 @@ export enum ViewerActionTypes {
   editNew = 'editNew',
 }
 
+/**
+ * * @private
+ * Hide from docs.
+ * ------
+ * User setting changes that can be subscribed to,
+ */
+export enum UserSettingKeys {
```

Comment:
May be we just say UserSettingTypes ? Keys looks very technical here.

Reply:
Changed to UserSettingTypes.